lib/template.js

```diff
@@ -1,14 +1,16 @@
 var template = require('lodash.template');
-var reInterpolate = require('lodash._reinterpolate');
+var reEscape = require('lodash.reescape');
```

Comment:
Sorry about this the exposed `lodash.reescape`, `lodash.reevaluate`, `lodash.reinterpolate` were a mistake. You can reference their former `lodash._reescape`, `lodash._reevaluate` `lodash._reinterpolate` forms.